styles/helpers/_lists.scss

```diff
@@ -1,3 +1,7 @@
 .h-list-style-type-none {
   list-style-type: none;
 }
+
+.h-list-style-type-disc {
```

Comment:
Would it be better to use `generate-helper` here and have a helper ready for all list styles?

Reply:
I did take a quick look at this, and the code we would have ended up with would have looked fairly similiar to this
```
/* ==========================================================================
   Transform - Rotate
   ========================================================================== */

$props: ('transform-rotate');
$subProps: ('');
@include generate-helper($props, $subProps) using ($selector, $property, $style) {
  #{$selector}{
    transform: rotate(0turn);
  }

  #{$selector}-quarter {
    transform: rotate(0.25turn);
  }

  #{$selector}-half {
    transform: rotate(0.5turn);
  }

  #{$selector}-three-quarters {
    transform: rotate(0.75turn);
  }
}
```
(☝️code I wrote) which when you actually think about it, is a very over engineered way of writing
```
  transform-rotate {
    transform: rotate(0turn);
  }

  transform-rotate-quarter {
    transform: rotate(0.25turn);
  }

  transform-rotate-half {
    transform: rotate(0.5turn);
  }

  transform-rotate-three-quarters {
    transform: rotate(0.75turn);
  }
```
so in this instance I avoided it, but we could write a bette helper for these cases